docs/src/main/asciidoc/security-protect-a-service-application-by-using-oidc-bearer-authentication-tutorial.adoc

```diff
@@ -0,0 +1,326 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+[id="security-protect-a-service-application-by-using-oidc-bearer-authentication-tutorial"]
```

Comment:
@rolfedh - While the name of this file matches the content, it's quite long.  Could this be reduced? i.e. remove article 'a' and 'by using'. 

Could we could abbreviate the topic type part of the templates and other dependencies in the framework to help reduce the size, i.e con for the concept, how for how-to, tut for tutorial and ref for reference?

Reply:
@ebullient  I think our tooling requires the full suffix. For example, it might require `-tutorial` rather than `-tut` for the front page to sort topics by content type.

Reply:
Thanks for the comment and reminder of our conversation, @michelle-purcell. I've shortened it to `security-protect-service-application-using-oidc-bearer-authentication-tutorial`.

Reply:
@rolfedh - Looks good. Thanks.  Yes, we can't change the ending bit yet... this would need some doc build and landing page architecture discussion + work, which is probably outside the scope of this PR.